e107_plugins/forum/forum_post.php

```diff
@@ -859,6 +859,11 @@ function renderForm()
 		$sc         = e107::getScBatch('post', 'forum')->setScVar('forum', $this->forumObj)->setScVar('threadInfo', vartrue($data))->setVars($data);
 		$text       = e107::getParser()->parseTemplate($template['form'], true, $sc);
 
+		if(!empty($template['caption']))
+		{
+      			$this->forumObj->prefs->set('title', e107::getParser()->parseTemplate($template['caption'], true, $sc));
+    		}
+
 		$this->render($text);
```

Comment:
Yes, i have donne it there because i got an empty $template variable if i try to get the template again before the table render inside [function render](https://github.com/rica-carv/e107/blob/58009dd3c54c353808d55617506a7177e52fbfd6/e107_plugins/forum/forum_post.php#L969)

If you have a workaround....